test/StreetNameRegistry.Tests/AggregateTests/WhenCorrectingHomonymAdditions/GivenHomonymAdditionForStreetNameAlreadyExists.cs

```diff
@@ -0,0 +1,90 @@
+namespace StreetNameRegistry.Tests.AggregateTests.WhenCorrectingHomonymAdditions;
+
+using AutoFixture;
+using Be.Vlaanderen.Basisregisters.AggregateSource.Testing;
+using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+using global::AutoFixture;
+using Municipality;
+using Municipality.Commands;
+using Municipality.Events;
+using Municipality.Exceptions;
+using Testing;
+using Xunit;
+using Xunit.Abstractions;
+
+public class GivenHomonymAdditionForStreetNameAlreadyExists : StreetNameRegistryTest
+{
+    private readonly MunicipalityStreamId _streamId;
+
+    public GivenHomonymAdditionForStreetNameAlreadyExists(ITestOutputHelper testOutputHelper) : base(testOutputHelper)
+    {
+        Fixture.Customize(new WithFixedMunicipalityId());
+        Fixture.Customize(new WithFixedPersistentLocalId());
+        Fixture.Customize(new InfrastructureCustomization());
+
+        _streamId = Fixture.Create<MunicipalityStreamId>();
+    }
+
+    [Fact]
+    public void GivenAStreetNameWithTheSameHomonymAdditionAlreadyExists_Throws_StreetNameNameAlreadyExistsException()
+    {
```

Comment:
Both the class and the method start with `Given`. Can we rename the method to something else? I think it suffices to just leave ThenThrowsStreetNameNameAlreadyExistsException